### PR TITLE
Add Debian 12 bookworm support; drop Debian 10 buster support

### PR DESCRIPTION
**Pull request by nevart**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -19,4 +19,4 @@ jobs:
     name: Integration
     uses: systemli/github-ansible-workflow/.github/workflows/ansible-integration-workflow.yaml@main
```
Comment (t2d) [inline .github/workflows/main.yml:20]:
```suggestion
    uses: systemli/github-ansible-workflow/.github/workflows/ansible-integration-workflow.yaml@v1.2.0
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -19,4 +19,4 @@ jobs:
...
     with:
-      distros: '[ "debian11", "debian10" ]'
+      distros: '[ "debian12", "debian11" ]'
```
Comment (t2d) [inline .github/workflows/main.yml:22]:
```suggestion
```
this is already the default

**Review by t2d — CHANGES_REQUESTED** [review]
The most interesting part will be if `actionable.py` will still work with the new version of ansible in bookworm. It was already very hacky in the past. Have you tried that?

File: .github/workflows/main.yml (hunk 2 above)
Comment (t2d) [reply .github/workflows/main.yml:22]: yes :)

**Comment by 0x46616c6b** [comment]
I removed the `actionable` callback plugin in #29 because it is deprecated and we can use the default callback with a specific option (already applied in #29)

**Review by 0x46616c6b — COMMENTED** [review]
It seems that the defaults are not working as expected. Without setting them explicitly here, it will be tested against Debian 10 and Debian 11.

**Pull request merged**